rosdep/base.yaml

```diff
@@ -2891,7 +2897,7 @@ libvtk-qt:
     stretch: [libvtk6-qt-dev]
     wheezy: [libvtk5-qt4-dev]
   fedora: [vtk-qt]
-  gentoo: [sci-libs/vtk]
+  gentoo: ['sci-libs/vtk[qt4]'] #qt5 is available too
```

Comment:
I don't think we should put a `USE` flag here, should the user have enabled/disabled `qt4` or `qt5`.